1.segment-data/README.md

```diff
@@ -0,0 +1,90 @@
+# Segment Cell Painting Images
+
+In this module, we present our pipeline for segmenting nuclei and cyoplasm from the Cell Health data.
+
+### Segmentation
+
+We use the CellPose segmentation algorithm to segment the nuclei from the Hoechst channel of each Cell Painting image.
+CellPose was first introduced in [Stringer, C., Wang, T., Michaelos, M. et al., 2020](https://doi.org/10.1038/s41592-020-01018-x) and we use the [python implementation](https://github.com/mouseland/cellpose).
+
+Stringer et al. trained the CellPose segmentation models on a diverse set of cell images and is therefore a good selection for our use case.
+The CellPose python implementation was particularly useful for building reproducible pipelines.
+
+### Nuclei segmentation
+
+After manually experimenting with CellPose on about 10 Cell Health nuclei images (various cell lines), we settled on the following parameters for CellPose nuclei segmentation:
+- `model_type = "cyto"` This parameter forces CellPose to use the cytoplasm model, which we found segments nuclei in the Cell Health data significantly better than the nucleus model. 
+More information about CellPose models can be found at https://cellpose.readthedocs.io/en/latest/models.html.
+- `channels = [0,0]` This parameter forces the model to segment cells in grayscale (in the case of Cell Health data single channel images).
+- `diameter = 80` This parameter indicates to the model that the average cell diameter is 80 pixels.
+- `flow_threshold=0` This paramenter decreases the maximum allowed error of the flows for each mask (default is `flow_threshold=0.4`).
+- `cellprob_threshold=0` This parameter is used to determine ROIs (default is `cellprob_threshold=0`).
+
+### Cytoplasm segmentation
+
+After manually experimenting with CellPose on about 10 Cell Health cytoplasm images (various cell lines), we settled on the following parameters for CellPose cytoplasm segmentation:
+- `model_type = "cyto"` This parameter forces CellPose to use the cytoplasm model for segmentation.
+More information about CellPose models can be found at https://cellpose.readthedocs.io/en/latest/models.html.
+- `channels = [1,3]` This parameter forces the model to segment cells using the nuclei and RNA channels of cell painting images. 
+We overlay nuclei, ER, and RNA channels for CellPose with our `overlay_images()` function in [segment-cell-health-data.ipynb](segment-cell-health-data.ipynb).
+
+**Note:** The channel numbers 1,3 correspond to the CellPose colors red and blue respectively.
```

Comment:
Great to see this elaboration - can you add a bit more about ER? This is a non-standard approach (innovative!) and I think it's worth mentioning a bit more detail (I don't see ER mentioned in the note)